.github/workflows/PR_Title_Check.yml

```diff
@@ -0,0 +1,74 @@
+name: Pull Request Title Checker
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Controls when the workflow will run
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize, reopened]
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+permissions:
+  pull-requests: read
+
+env:
+  MSDK_DIR: msdk
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # The verify job will check the differences with any updated register file.
+  check:
+    # The type of runner that the job will run on
+    runs-on: [ ubuntu-latest ]
+
+    steps:
+      - name: Checking PR Title
+        uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+        with:
+          types: |
+            fix
+            feat
+            chore
+            refactor
+            docs
+            style
+            test
+            perf
+            ci
+            build
+            revert
+
+          scopes: |
+            Documentation
+            Examples
+            Tools
+            BLE
+            Boards
+            CMSIS
+            MiscDrivers
+            PeriphDrivers
+            Third-Party
+            ignore
+            workflow
+            Other
+
```

Comment:
If these terms are going to end up in the release notes, I'd like to use "ThirdParty", "Ignore", and "Workflow" to maintain consistent capitalization and punctuation.

Reply:
To clarify, I'm suggesting removing the '-' from "Third-Party", and capitalizing the first letter of "ignore" and "workflow".